Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -268,6 +262,17 @@ def examineFiles(self,files):
     for loadable in loadables:
       loadable.files, distances, loadable.warning = DICOMUtils.getSortedImageFiles(loadable.files, self.epsilon)
 
+    # by default, prefer the all-files loadable
+    loadables[0].confidence = .55
+    if subseriesCount == 1 and loadables[0].warning != "":
```

Comment:
We may have more subseries if "allowLoadingByTime" is set to True. This means that if the user allows loading subseries by time then it may seemingly break loading of volumes that were previously loaded correctly (split based on AcquisitionNumber). To prevent this, what do you think about putting subseries keys in decreasing priority order (I think the current order would be correct), and using the _first_ set of split subseries (corresponding to the highest priority key)?

Reply:
That could work, but wouldn't it be better to generate a sequence if we detect a subseries that is based on time?

On a similar note, I'm thinking that it's pretty awkward to have one checkbox per subseries in the advanced panel of the browser - makes it hard to switch between choices.  Perhaps we should make one compound loadable that represents a subseries option.  Or maybe the subseries should always be sequences with the frames flagged by the split value.

Reply:
One checkbox per subseries might be useful in some cases (to be able to load particular time points or orientations), but I agree that it can be quite inconvenient, especially when it is more than a few nodes. I like both ideas - compound loadable and sequence. Since it is easier to split a sequence to nodes than construct a sequence from a set of nodes, and sequences work much better when there are more than a few items, I think it would be better to always load a subseries as a sequence.

Reply:
Do you have a sequence importer already?  The DICOMVolumeSequencePlugin appears to only do export.

Reply:
Yes, the sequence importer is in https://github.com/fedorov/MultiVolumeImporter/blob/master/MultiVolumeImporterPlugin.py

In fact, this importer already splits a series into subseries based on selected tags. Maybe we should just remove splitting by subseries from scalar volume plugin and let the multivolume importer handle this?

Reply:
Yes, I like that in general for things that are naturally sequential like perfusion.  I notice that MultiVolumeImporter doesn't use AcquisitionNumber to detect sequences, and that may be okay.  The lung CT data has different acquisitions that correspond to different parts of the body at different FOVs so maybe more natural as multiple scalar volumes.  So I think we should just wrap this up and leave it for now, but also make sure the confidence for Sequence loadables is higher for any of the time-related subseries.

By the way, testing the MultiVolumeImporter it generates this message at the end (loading works because it's completed before this error).


```
DICOM plugin failed to load 'dMRI_dir107_PA - as a 108 frames Volume Sequence by ImagePositionPatient+InstanceNumber' as a 'MultiVolume'.
Traceback (most recent call last):
  File "/home/researcher/Downloads/Slicer-4.11.0-2020-04-19-linux-amd64/bin/../lib/Slicer-4.11/qt-scripted-modules/MultiVolumeImporterPlugin.py", line 700, in load
    sequenceBrowserModule = slicer.modules.sequencebrowser
AttributeError: module 'modules' has no attribute 'sequencebrowser'

```
I'm guessing you know an easy fix for this?
https://github.com/fedorov/MultiVolumeImporter/blob/master/MultiVolumeImporterPlugin.py#L699

Reply:
Moved this to a new issue: https://github.com/fedorov/MultiVolumeImporter/issues/41

I'll go ahead and merge this, since we know it addresses an issue in the current master.  We can open something new if more corner cases come up.